user/nfpcore/rte_config.h

```diff
@@ -0,0 +1,123 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
We don't need this file as we only support 1 configuration. We can just remove the includes from other headers.